src/DynamoCoreWpf/UI/Themes/Modern/Ports.xaml

```diff
@@ -377,15 +364,15 @@
                                                                Grid.Column="0"
                                                                Width="14px"
                                                                Height="14px"
-                                                               Stroke="{StaticResource PortUseLevelsCheckBoxColor}"
-                                                               StrokeThickness="2" />
+                                                               Stroke="#C0C0C0"
```

Comment:
Should we also define this color somewhere?

Reply:
Done - Changed to the very similar PrimaryCharcoal300Brush